ambry-router/src/main/java/com/github/ambry/router/ParanoidDurabilityOperationTracker.java

```diff
@@ -250,7 +252,21 @@ private boolean hasSucceededLocally() {
    * @return true if we have received enough successful responses from remote replicas, false otherwise.
    */
   private boolean hasSucceededRemotely() {
-    return remoteReplicaSuccessCount >= remoteReplicaSuccessTarget;
+    if (remoteReplicaSuccessCount >= remoteReplicaSuccessTarget) {
+      return true;
+    }
+    else if(remoteAttempts >= remoteAttemptLimit) {
```

Comment:
the indentation is a bit weird, it should be
```
} else if (xxx) {
}
```
could you please format this file?

Reply:
Yeah, thanks for catching this - it's fixed.